test/import_resource_test.go

```diff
@@ -0,0 +1,209 @@
+package test
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/instaclustr/terraform-provider-instaclustr/instaclustr"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestAccCluster_importBasic(t *testing.T) {
```

Comment:
We should have multiple test files for each of the offerings (C*, Kafka, Redis) and do tests for each one as they use different bundle configurations and bundle specific schema sections. These should be covered by tests.